mono_repo/lib/src/commands/travis.dart

```diff
@@ -57,7 +57,7 @@ class CIConfigMismatchException extends UserException {
 
 /// Checks that the root CI config includes exactly one provider, travis.
 void _checkCIConfig(RootConfig rootConfig) {
-  if (rootConfig.monoConfig.ci != [CI.travis]) {
+  if (rootConfig.monoConfig.ci != {CI.travis}) {
```

Comment:
Is this doing what you want it to? Should this be using `SetEquality` from `package:collection`?

Reply:
Yeah...I was wondering that when I refactored this...hrm...

Reply:
fixed

Reply:
right, @jakemac53 ?